### PR TITLE
PA-1514 change layer popup contents

### PR DESCRIPTION
**Pull request by goemen**
After a meeting with Mikaela and Lindsay, we came up with a list of fields to be displayed in the layer popup. This ticket changes the layer popup content to match the requirements. 

Also, I noticed that we have added lots of styled-components and the tests did not include/import the jest-styled-components for consistent classNames. I imported the `jest-styled-components` globally in the test setup to address that.

### CODE REVIEW

File: frontend/src/features/admin/users/__snapshots__/ManageUsers.test.tsx.snap [hunk 1]
```diff
@@ -224,7 +229,7 @@ exports[`Manage Users Component Snapshot matches 1`] = `
     </div>
   </form>
   <div
-    className="sc-fzolEj kuDfzp container-fluid"
+    className="c0 container-fluid"
```
Comment (Fosol) [inline frontend/src/features/admin/users/__snapshots__/ManageUsers.test.tsx.snap:232]: Why are you changing the class name?

File: frontend/src/setupTests.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
+import 'jest-styled-components';
```
Comment (Fosol) [inline frontend/src/setupTests.ts:6]: Awesome!

**Review by Fosol — APPROVED** [review]

**Review by jdtoombs — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

**Review by LawrenceLau2020 — APPROVED** [review]
Looks good!

**Review by agalbrai — APPROVED** [review]

**Review by Bruce451 — APPROVED** [review]
Useful comments here.

**Pull request merged**